while-loops/while-loops.md

```diff
@@ -0,0 +1,257 @@
+# While Loops
+
+## Learning Goals
+
+* Practice using while loops to loop while a conditional is true 
+* Apply a counter variable while using while loops
+
+## Introduction
+
+Imagine writing software that creates digital group greeting cards. Before the user can sign a digital group greeting card, they need to type in the specific password `AdaLovelaceCodesIt` . If they don't type in the correct password, then the app should ask them to type in the password again, and again, and again, until the user leaves or types in `AdaLovelaceCodesIt` .
+
+How can we solve this problem using Python code?
+
+for loops allow programmers to iterate over lists. Sometimes, we need to repeat lines of code, but not because we are iterating through a list. We can explore the while loop as an alternative!
+
+## Vocabulary and Synonyms
```

Comment:
"terminology" and "term" tend to be more often used to refer to words and phrases in the context of a particular field e.g. computer science.

Reply:
Good thought-- This header is "good enough" and it's in enough lessons (😆 ) that an update to the header should happen curriculum-wide